finetune/lm_base.py

```diff
@@ -100,17 +97,18 @@ def get_target_encoder(self):
         else:
             return RegressionEncoder()
 
-    def _finetune(self, *Xs, Y, batch_size=BATCH_SIZE, val_size=0.05, val_interval=150, val_window_size=5):
+    def _finetune(self, *Xs, Y, batch_size=None, val_size=0.05, val_interval=150, val_window_size=5):
```

Comment:
val_size and val_interval and val_window_size should have corresponding hparam options